### DISCUSSION TITLE
Displaying long strings in table headers

### DISCUSSION THREAD

**Discussion by Fazle-Jahangir** [twilio-labs/paste#3015] [General] [answered]
[Redacted]

Non-public information

**Comment by loreina** [comment, marked as answer]
Hi @Fazle-Jahangir,

We'd recommend not hiding the column header and letting the header text wrap so users are aware of the full text since it's labelling the column. As suggested in #2950, we'd encourage using Data Grid here as the data is quite dense. Thanks!
